### PR TITLE
EOS-17576: cleanup requirement file for LR2

### PR DESCRIPTION
**Pull request by azheregelya**
Removed packages are not used by HA in LR2.

Signed-off-by: Andrei Zheregelia <andrei.zheregelia@seagate.com>

## Problem Statement
<pre>
  <code>
  Story Ref (if any): https://jts.seagate.com/browse/EOS-17576
   Cleanup requirement file for LR2
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
On VM.
  </code>
</pre>
## Problem Description
<pre>
  <code>
Packages listed in jira ticket are not used by HA in LR2.
  </code>
</pre>
## Solution
<pre>
  <code>
Remove.
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
N/A
  </code>
</pre>

### CODE REVIEW

**Review by indrajitzagade — APPROVED** [review]

**Review by indrajitzagade — COMMENTED** [review]
Lets get the UT results

**Comment by azheregelya** [comment]
Ready for merge.

File: jenkins/cicd/cortx-ha-dep.sh [hunk 1]
```diff
@@ -61,7 +61,7 @@ if [ "$DEV" == "false" ]; then
     then
         req_file=${BASE_DIR}/jenkins/pyinstaller/v1/requirements.txt
     else
-        req_file=${BASE_DIR}/jenkins/pyinstaller/v2/requirements.txt
+        req_file=${BASE_DIR}/requirements.txt
```
Comment (ajaysrivas) [inline jenkins/cicd/cortx-ha-dep.sh:64]: We had decide a directory structure for such files. Let's keep the directory structure intact. Please revert the path.

**Review by ajaysrivas — CHANGES_REQUESTED** [review]

**Comment by ajaysrivas** [comment]
@ajay-paratmandali @indrajitzagade Please review the packages in requirement.txt. I guess that there are still some packages which we are not using directly. For example - elasticsearch.

**Review by ajay-paratmandali — APPROVED** [review]

**Comment by azheregelya** [comment]
Removed: 
-aiohttp==3.6.1                                                                                                                                                                                                   
-paramiko==2.7.1                                                                                                                                                                                                                                                                                                                                                                                             
-prettytable==0.7.2                                                                                                                                                                                               
-pika==1.1.0                                                                                                                                                                                                      
-idna==2.8                                                                                                                                                                                                        
-elasticsearch>=6.0.0,<7.0.0                                                                                                                                                                                      
-elasticsearch-dsl==6.4.0

Other modules are either used in tests or in build process.
Tested with jenkins/build.sh script.

**Comment by azheregelya** [comment]
@ajay-paratmandali @indrajitzagade @ajaysrivas, do we have a reason to freeze argparse version in requirements.txt ?

File: jenkins/cicd/cortx-ha-dep.sh (hunk 1 above)
Comment (azheregelya) [reply jenkins/cicd/cortx-ha-dep.sh]: Reverted.

**Review by ajaysrivas — APPROVED** [review]

**Pull request merged**